source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -21,6 +21,7 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "BioXASImagingAppController.h"
 
+#include "beamline/CLS/CLSBeamlines.h"
```

Comment:
if you take a look at the first commit only, you might understand. I was trying to put our Beamline definitions in a single file, so that we won't have duplicated definitions in the future anymore.